core/jobs_registry.py

```diff
@@ -40,7 +41,8 @@
     exp_jobs_one_off.ExplorationContributorsSummaryOneOffJob,
     email_jobs_one_off.EmailHashRegenerationOneOffJob,
     user_jobs_one_off.UserProfilePictureOneOffJob,
-    exp_jobs_one_off.ItemSelectionInteractionOneOffJob]
+    exp_jobs_one_off.ItemSelectionInteractionOneOffJob,
+    user_query_jobs_one_off.UserQueryOneOffJob]
```

Comment:
The admin wouldn't run this, right? So, no need to show in the jobs dashboard, I think?